eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -248,21 +248,24 @@ object Graph {
           val neighbor = edge.desc.a
           // NB: this contains the amount (including fees) that will need to be sent to `neighbor`, but the amount that
           // will be relayed through that edge is the one in `currentWeight`.
```

Comment:
This comment is now misplaced, can you put it right before `val neighborWeight = ...`?